kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -235,7 +235,10 @@ define([
                         );
                     collapsedCellJobStatus = Jobs.createJobStatusFromFsm(fsmMode, fsmStage);
                 } else if (utils.getCellMeta(_cell, 'kbase.bulkImportCell.exec.jobs')) {
-                    const currentJobs = utils.getCellMeta(_cell, 'kbase.bulkImportCell.exec.jobs');
+                    const currentJobs = utils.getCellMeta(
+                        _cell,
+                        'kbase.bulkImportCell.exec.jobs.byId'
+                    );
```

Comment:
example of where the functions changed above are used -- change the input to match the new code